apps/admin/frontend/package.json

```diff
@@ -86,12 +86,14 @@
     "normalize.css": "^8.0.1",
     "pagedjs": "^0.1.40",
     "path": "^0.12.7",
+    "pdfjs-dist": "3.11.174",
```

Comment:
We need to serve the frontend a version of the `pdf.worker` file, from `pdfjs-dist`. The recommended `react-pdf` way wasn't working for me, so I did what @eventualbuddha had previously done.

We can't use the `pdfjs-dist` version installed in other parts of the repo, because the version mismatch with `react-pdf`'s version causes the worker to fail. So I installed `pdfjs-dist` here in `admin/frontend`, with a version matching `react-pdf`. But now I'm running afoul of monorepo validation. @eventualbuddha is there a blacklist for that?

We could upgrade `pdfjs-dist` across the repo but I haven't scoped that out, it's a major version change.

Alternatively, we could just try to serve the `pdf.worker.min.js` file out of the public folder. It would be copy-pasta'd, but it might be easier than explicitly hosting it from the prodserver.